setup.py

```diff
@@ -73,7 +73,7 @@
     "mrcz": ["blosc>=1.5", 'mrcz>=0.3.6'],
     "test": ["pytest>=3", "pytest-mpl", "matplotlib>=2.0.2"],
     "doc": ["sphinx>=1.7", "sphinx_rtd_theme"],
-    "speed": ["numba<0.39"],
+    "speed": ["numba"],
 
 }
 extras_require["all"] = list(itertools.chain(*list(extras_require.values())))
```

Comment:
I think that we should remove `"test"` from `"all"`, because there is no reason to install pytest and pytest-mpl for an user install.

Reply:
I agree.

Reply:
Why has this gone? I thought that it was a nice idea.

Reply:
This is what was breaking the installation in python 3.5. I couldn't figure out why and what is happening.... I would not have time to finish this, so this is why I reverted it.